.circleci/config.yml

```diff
@@ -54,14 +55,20 @@ jobs:
       - run:
           name: Install dependencies
           command: |
-            pip install -U virtualenv --user
+            python -m pip install --progress-bar off -U virtualenv --user
             mkdir -p ./venv
-            python -m virtualenv ./venv
+            python -m virtualenv ./venv --clear
             . venv/bin/activate
-            python -m pip install --progress-bar off ${SIGNAC_VERSION}
-            python -m pip install --progress-bar off -r requirements.txt
-            python -m pip install --progress-bar off -r requirements-test.txt
+            python -m pip install --progress-bar off -U pip>=20.3
             python -m pip install --progress-bar off -U codecov
+            # DEPENDENCIES can be "OLDEST", or "NEWEST"
+            if [ "${DEPENDENCIES}" == "OLDEST" ]; then
+              python -m pip install --progress-bar off -r .circleci/ci-oldest-reqs.txt
+            else
+              python -m pip install --progress-bar off ${SIGNAC_VERSION}
```

Comment:
I'd like to delete the pinning for `signac` from the "oldest" requirements and let that be handled by the `${SIGNAC_VERSION}` variable.

Move this line outside the "if", install `signac` before or after the rest of the dependencies (this should run for both OLDEST and NEWEST).